packages/api-request-builder/src/build-query-string.js

```diff
@@ -16,12 +16,35 @@ export default function buildQueryString (
   if (!params)
     throw new Error('Missing options object to build query string.')
 
-  const { query, pagination, search, expand, staged, searchKeywords } = params
+  const {
+    query,
+    pagination,
+    search,
+    expand,
+    staged,
+    priceCurrency,
+    priceCountry,
+    priceCustomerGroup,
+    priceChannel,
+    searchKeywords,
```

Comment:
I tried doing this and not sure how the naming of the group will be like, aside prices selection that is a bit obvious, not sure any other can be group